packages/ui/src/app/pages/Election/BlacklistedAccounts/BlacklistedAccounts.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import { BN_THOUSAND } from '@polkadot/util'
+import { Args, Meta, Story } from '@storybook/react'
+import BN from 'bn.js'
+import React from 'react'
+
+import { BalancesContext } from '@/accounts/providers/balances/context'
+import { AddressToBalanceMap } from '@/accounts/types'
+import { BN_ZERO } from '@/common/constants'
+import { MocksParameters } from '@/mocks/providers'
+
+import { BlacklistedAccounts } from './BlacklistedAccounts'
+
+const allAccounts = [
+  'j4Rc8VUXGYAx7FNbVZBFU72rQw3GaCuG2AkrUQWnWTh5SpemP',
+  'j4RjraznxDKae1aGL2L2xzXPSf8qCjFbjuw9sPWkoiy1UqWCa',
+  'j4RbTjvPyaufVVoxVGk5vEKHma1k7j5ZAQCaAL9qMKQWKAswW',
+  'j4Rh1cHtZFAQYGh7Y8RZwoXbkAPtZN46FmuYpKNiR3P2Dc2oz',
+]
+
+const mockDefaultBalance = {
+  locked: BN_ZERO,
+  recoverable: BN_ZERO,
+  transferable: BN_ZERO,
+  locks: [],
+  vestingTotal: BN_ZERO,
+  vestedClaimable: BN_ZERO,
+  vestedBalance: BN_ZERO,
+  vestingLocked: BN_ZERO,
+  vesting: [],
+  total: BN_THOUSAND,
+}
+
+const useMyBalances: AddressToBalanceMap = {
+  [allAccounts[0]]: {
+    ...mockDefaultBalance,
+    total: new BN(200_000_000_000),
+  },
+  [allAccounts[1]]: {
+    ...mockDefaultBalance,
+    total: new BN(200_000_000_000),
+  },
+  [allAccounts[2]]: {
+    ...mockDefaultBalance,
+    total: new BN(200_000_000_000),
+  },
+  [allAccounts[3]]: {
+    ...mockDefaultBalance,
+    total: new BN(200_000_000_000),
+  },
+}
+export default {
+  title: 'Pages/Blacklisted Accounts',
+  component: BlacklistedAccounts,
+  parameters: {
+    mocks: (): MocksParameters => {
+      return {
+        chain: {
+          derive: {
+            balances: {
+              all: {},
+            },
+          },
```

Comment:
> Are you sure this is needed ?

This is the error that shows up if i don't include it 
![image](https://github.com/Joystream/pioneer/assets/33874323/a73752d1-bb43-4518-b15b-df5446a4e385)

I think its because the `useBalance` hook is called which requires the `all()` 
![image](https://github.com/Joystream/pioneer/assets/33874323/9fee52be-8ebd-44b8-bf6e-57a19aa6491c)